targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/device/TOOLCHAIN_GCC_ARM/MK66FN2M0xxx18.ld

```diff
@@ -195,6 +196,18 @@ SECTIONS
     . = ALIGN(8);
     __interrupts_ram_end__ = .; /* Define a global symbol at data end */
   } > m_data
+  
+  .crash_data_ram :
+  {
+    . = ALIGN(4);
```

Comment:
For ARMC6 support in future we have updated all linker scripts to be 8-byte aligned.